### PR TITLE
Rework existing ci file

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
Depending on the outcome of https://github.com/doctrine/deprecations/pull/59, I will add `test_fixtures` to the cs job as well.

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name: "Coding Standards"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/coding-standards.yml
+      - composer.*
+      - src/**
```
Comment (SenseException) [inline .github/workflows/coding-standards.yml]:
```suggestion
      - lib/**
```

File: .github/workflows/coding-standards.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - phpcs.xml.dist
+      - test_fixtures/**
+      - tests/**
+  push:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/coding-standards.yml
+      - composer.*
+      - src/**
```
Comment (SenseException) [inline .github/workflows/coding-standards.yml]:
```suggestion
      - lib/**
```

File: .github/workflows/continuous-integration.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/continuous-integration.yml
+      - composer.*
+      - src/**
```
Comment (SenseException) [inline .github/workflows/continuous-integration.yml]:
```suggestion
      - lib/**
```

File: .github/workflows/continuous-integration.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      - phpunit.xml.dist
+      - test_fixtures/**
+      - tests/**
+  push:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/continuous-integration.yml
+      - composer.*
+      - src/**
```
Comment (SenseException) [inline .github/workflows/continuous-integration.yml]:
```suggestion
      - lib/**
```

**Review by SenseException — CHANGES_REQUESTED** [review]

**Comment by greg0ire** [comment]
@SenseException nice catch!

**Review by derrabus — APPROVED** [review]

**Pull request merged**